### PR TITLE
fix: populate-user-agent-header

### PR DESCRIPTION
**Pull request by Misieq01**
### [ Content ]
Added User-agent header to request that had it missing or was not descriptive

### CODE REVIEW

File: src-tauri/src/binaries/adapter_tor.rs [hunk 1]
```diff
@@ -71,7 +71,14 @@ impl LatestVersionApiAdapter for TorReleaseAdapter {
             .join(format!("{}.asc", asset.name));
         let checksum_url = format!("{}.asc", asset.url);
 
-        match download_file_with_retries(&checksum_url, &checksum_path, progress_tracker).await {
+        let user_agent = format!(
+            "universe {}({}) | {}",
+            env!("CARGO_PKG_VERSION"),
+            std::env::consts::OS,
+            "github adapter"
```
Comment (brianp) [inline src-tauri/src/binaries/adapter_tor.rs:78]: Should this say "tor adapter" ?

```suggestion
            "tor adapter"
```

**Review by brianp — CHANGES_REQUESTED** [review]
I don't think we need different user-agents per adapter github/tor/binary download. All the agents could just say universe with version and platform, and if this is the case it could be a static var instead of defined per file.

/cc @leet4tari opinions?

File: src-tauri/src/binaries/adapter_tor.rs (hunk 1 above)
Comment (leet4tari) [reply src-tauri/src/binaries/adapter_tor.rs:78]: Agree, only TU version and platform info I think is useful. Getting reports on TU version or platform usage from CDN provider is possible. Different ```adapter``` would not request resources from different services.

**Comment by Misieq01** [comment]
Closing in favor of https://github.com/tari-project/universe/pull/935

**Pull request closed without merging**